components/fieldgroups/conversion/exchangerate-details.schema.json

```diff
@@ -0,0 +1,83 @@
+{
+  "meta:license": [
+    "Copyright 2022 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/exchangerate-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Exchange Rate Details",
```

Comment:
Can we call it as "Currency Conversion Rate Details" ? exchange rate itself could be very generic.